public/css/module.less

```diff
@@ -12,33 +12,29 @@ a:focus {
     align-items: center;
     font-size: 1.3em;
```

Comment:
Removing the font-size Parameter here, makes the font the same size as the label's.

Also It's the same font-size, as everywhere else in the UI viewport (as well as in the quick-action bar in Icinga DB Web's Detail view, where the action buttons even are the same color).

So I don't see why this would be too small. What I see is that it makes the font and element sizes in the control area more consistent.

As an additional effect it makes the controls less distracting from the actual content (the process nodes), which should be where the user is mostly focussed on.

Look at it here.
![Screenshot 2024-03-28 at 17 35 49](https://github.com/Icinga/icingaweb2-module-businessprocess/assets/6977434/7ee57216-99ed-4fa2-a8f1-51abd318dcc8)

Now the only thing that's off, is the breadcrumb element. If you want, you can align that as well, though I don't see that as a priority.